docs/22_county_runbook.md

```diff
@@ -32,7 +32,7 @@ text of the Run Book.
 A County user can launch the RLA Tool by pointing a browser to the
 URL provided by the Colorado Department of State.
 
-![Login](./screenshots/100_login_screen_1.png)
+![Login](./screenshots/v1a1_100_login_screen_1.pdf)
```

Comment:
I'm suggesting that the `v1a1_` version number be left out of the file name.
I also see now that the replacement screenshot is a pdf rather than a png file. I would think offhand that we wouldn't want to mess with pdf files for screenshots, but perhaps there is a good reason for this one and `203_County_Home_Mid_Audit.pdf` to be pdfs?
Sorry if I'm just missing the basic approach here....

Reply:
JoeK wants pdf images, says they reproduce better. We're in the process of replacing all other file types with pdfs.

Version number prefixes help us by grouping files nicely in alphabetic order.

Reply:
I'm not a GitHub expert, as you know. I made this pull request to honor the value of merging branches in fairly regularly. 

I want to make sure that as @mkiniry  works on screenshots and runbook/user manual text we don't get into version control hell. 

@nealmcb if you're not ready to approve this merge, please let me know what (if anything) I could do now to minimize rebasing, etc.  Otherwise I'll just let @mkinry take care of the documentation while I focus today on testing issues.

Reply:
Cool re: the pdfs.
Re the version numbers, I see that there are lots of multiple versions of screenshots in there now. I don't want to slow you down, and guess under the circumstances that is perhaps something to come back to later, since it just doesn't seem right to me to have multiple versions of the same page over time represented as different files, and storing old stuff in a `screenshots/000\ Old\ Screenshots/` directory.
@ranweiler, our git expert, raised the "empty commit" issue but I'm not sure what he's referring to there. If that's clarified, I can help take out the commit that is bringing in the wrong version of 
`npm-shrinkwrap.json`, which needs to happen before merge.

Reply:
My recommendation to @mkiniry is that we keep screenshots for specific releases in a subdirectory named according to the release name and then simply overwrite the existing screenshot files with new versions.  This will permit us to compare/contrast different versions of the same screen across multiple releases without having to jump through hoops with git.  I definitely am in alignment with @nealmcb wrt not naming files with version numbers and that kind of thing.

Reply:
I also highlighted to @mkiniry that we need not use SVG/PDF screenshots for this upcoming release; PNGs from screenshots are fine.  But we will likely need them, or at least PNGs grabbed from a Mac with a Retina display, for the final release, otherwise the Run Book and User Manual will not look professional when printed.

Reply:
Sounds exactly right - thanks. We can use subdirectories now, learn git-fu and remove the subdirectories when appropriate.
Besides dropping the version numbers from individual files, can we also eliminate spaces from all file and directory names?